### ISSUE TITLE
module question dot causes loop

### ISSUE THREAD

**Issue by GanerCodes** [xonsh/xonsh#4729]
type in terminal:
```py
import os
os?. # it happens with any number of question marks
```
it will infinitely display the help message

**Comment by jnoortheen** [comment]
It is customary to be fill the xonfig output. So that we can later check when it appeared and fixed. Please add the info. Fixes are always welcome.

**Comment by GanerCodes** [comment]
+------------------+----------------------+
| xonsh            | 0.11.0               |
| Git SHA          | 337cf25a             |
| Commit Date      | Nov 17 15:37:41 2021 |
| Python           | 3.10.2               |
| PLY              | 3.11                 |
| have readline    | True                 |
| prompt toolkit   | 3.0.28               |
| shell type       | prompt_toolkit       |
| history backend  | sqlite               |
| pygments         | 2.11.2               |
| on posix         | True                 |
| on linux         | True                 |
| distro           | gentoo               |
| on wsl           | False                |
| on darwin        | False                |
| on windows       | False                |
| on cygwin        | False                |
| on msys2         | False                |
| is superuser     | False                |
| default encoding | utf-8                |
| xonsh encoding   | utf-8                |
| encoding errors  | surrogateescape      |
| on jupyter       | False                |
| jupyter kernel   | None                 |
| xontrib          | []                   |
| RC file 1        | /home/ganer/.xonshrc |
+------------------+----------------------+

**Comment by jnoortheen** [comment]
I am trying to reproduce this on v0.14.0 . Are you still facing this error?

**Comment by GanerCodes** [comment]
yep it was fixed at some point in the past couple updates, good work yall
